### PR TITLE
Jdt enhancements

### PR DESCRIPTION
**Pull request by scottresnik**
Just added a few convenience methods to enhance the DSL.

Feedback welcome.

### CODE REVIEW

File: src/main/java/com/diffplug/gradle/oomph/ConventionJdt.java [hunk 1]
```diff
@@ -33,24 +39,33 @@
...
  *         }
+ *         compilerComplianceLevel('1.6')
+ *         classpathVariable('myClasspath', '/var/lib/repo')
  *     }
  * }
  * ```
  */
 public class ConventionJdt extends OomphConvention {
+	final static String JDT_CORE_PREFS = ".metadata/.plugins/org.eclipse.core.runtime/.settings/org.eclipse.jdt.core.prefs";
+	final static List<String> JDT_COMPLIANCE_PROPS = Collections.unmodifiableList(Arrays.asList("org.eclipse.jdt.core.compiler.codegen.targetPlatform", "org.eclipse.jdt.core.compiler.compliance", "org.eclipse.jdt.core.compiler.source"));
```
Comment (nedtwigg) [inline src/main/java/com/diffplug/gradle/oomph/ConventionJdt.java]: This is a good usecase for `com.diffplug.common.collect.ImmutableList.of()`

Comment (nedtwigg) [inline src/main/java/com/diffplug/gradle/oomph/ConventionJdt.java]: But it's fine as-is :)  Is this about what you wanted to do?  I've got some thoughts on a way to reorganize this, but don't wanna cause a conflict.  Just lemme know when you're at a stopping point and I'll push it up.

Comment (scottresnik) [inline src/main/java/com/diffplug/gradle/oomph/ConventionJdt.java]: ImmutableList was the right way to go.  I pushed the change in.

You're free to do your worst now. :)

**Comment by nedtwigg** [comment]
I left my comments in the commit messages.  Take a look, and lemme know if it looks alright to you.  If it does, and I didn't break the CI, I'll merge it.  Thanks for another great contribution!!  You got other stuff coming down the pipe re: Goomph?

**Comment by scottresnik** [comment]
Changes look good.  #21 is the only thing I might look at, but I'm not real optimistic about getting to it until next week.

**Comment by nedtwigg** [comment]
Sounds good.

**Pull request merged**